vagrant/provision.sh

```diff
@@ -31,7 +31,7 @@ chmod 777 /var/maestro
 echo "#!/bin/bash
```

Comment:
```suggestion
echo "#!/bin/bash -ue
```

Add -u and -e flags. -u throws an error if a var is undefined, and -e exits on any error.

Reply:
https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html